mobile/src/bundles/common/styles/sizes.ts

```diff
@@ -16,6 +16,12 @@ const SIZES = {
     borderRadius5: {
         borderRadius: 5,
     },
+    borderRadius24: {
+        borderRadius: 24,
+    },
+    borderWidth2: {
+        borderWidth: 2,
+    },
```

Comment:
You can leave these styles in style.ts, since it's edge case for now related to only 1 button type

Reply:
should I create variables for this values or numbers will be ok?

Reply:
fix